pkg/scaling/scale_handler.go

```diff
@@ -624,5 +624,140 @@ func (h *scaleHandler) getScaledObjectState(ctx context.Context, scaledObject *k
+	case "elasticsearch":
+		return scalers.NewElasticsearchScaler(config)
+	case "etcd":
+		return scalers.NewEtcdScaler(config)
+	case "external":
+		return scalers.NewExternalScaler(config)
+	// TODO: use other way for test.
+	case "external-mock":
+		return scalers.NewExternalMockScaler(config)
+	case "external-push":
+		return scalers.NewExternalPushScaler(config)
+	case "gcp-pubsub":
+		return scalers.NewPubSubScaler(config)
+	case "gcp-stackdriver":
+		return scalers.NewStackdriverScaler(ctx, config)
+	case "gcp-storage":
+		return scalers.NewGcsScaler(config)
+	case "graphite":
+		return scalers.NewGraphiteScaler(config)
+	case "huawei-cloudeye":
+		return scalers.NewHuaweiCloudeyeScaler(config)
+	case "ibmmq":
+		return scalers.NewIBMMQScaler(config)
+	case "influxdb":
+		return scalers.NewInfluxDBScaler(config)
+	case "kafka":
+		return scalers.NewKafkaScaler(config)
+	case "kubernetes-workload":
+		return scalers.NewKubernetesWorkloadScaler(client, config)
+	case "liiklus":
+		return scalers.NewLiiklusScaler(config)
+	case "loki":
+		return scalers.NewLokiScaler(config)
+	case "memory":
+		return scalers.NewCPUMemoryScaler(corev1.ResourceMemory, config)
+	case "metrics-api":
+		return scalers.NewMetricsAPIScaler(config)
+	case "mongodb":
+		return scalers.NewMongoDBScaler(ctx, config)
+	case "mssql":
+		return scalers.NewMSSQLScaler(config)
+	case "mysql":
+		return scalers.NewMySQLScaler(config)
+	case "nats-jetstream":
+		return scalers.NewNATSJetStreamScaler(config)
+	case "neo4j":
+		return scalers.NewNeo4jScaler(ctx, config)
+	case "new-relic":
+		return scalers.NewNewRelicScaler(config)
+	case "openstack-metric":
+		return scalers.NewOpenstackMetricScaler(ctx, config)
+	case "openstack-swift":
+		return scalers.NewOpenstackSwiftScaler(ctx, config)
+	case "postgresql":
+		return scalers.NewPostgreSQLScaler(config)
+	case "predictkube":
+		return scalers.NewPredictKubeScaler(ctx, config)
+	case "prometheus":
+		return scalers.NewPrometheusScaler(config)
+	case "pulsar":
+		return scalers.NewPulsarScaler(config)
+	case "rabbitmq":
+		return scalers.NewRabbitMQScaler(config)
+	case "redis":
+		return scalers.NewRedisScaler(ctx, false, false, config)
+	case "redis-cluster":
+		return scalers.NewRedisScaler(ctx, true, false, config)
+	case "redis-cluster-streams":
+		return scalers.NewRedisStreamsScaler(ctx, true, false, config)
+	case "redis-sentinel":
+		return scalers.NewRedisScaler(ctx, false, true, config)
+	case "redis-sentinel-streams":
+		return scalers.NewRedisStreamsScaler(ctx, false, true, config)
+	case "redis-streams":
+		return scalers.NewRedisStreamsScaler(ctx, false, false, config)
+	case "selenium-grid":
+		return scalers.NewSeleniumGridScaler(config)
+	case "solace-event-queue":
+		return scalers.NewSolaceScaler(config)
+	case "stan":
+		return scalers.NewStanScaler(config)
+	default:
+		return nil, fmt.Errorf("no scaler found for type: %s", triggerType)
+	}
+	// TRIGGERS-END
```

Comment:
This code has been moved to the file scalers_builder.go. Remove it from here and add the new scaler there please 
CI checks are failing due to this indeed